### ISSUE TITLE
settings set fails to set lbryum_servers (CLI only)

### ISSUE THREAD

**Issue by tzarebczan** [lbryio/lbry-sdk#2213]
## The Issue
Work via Json + passing list. Might be a good time to rename it to wallet_servers too. 

```
C:\Users\thoma\Desktop>lbrynet settings set lbryum_servers ['lbryumx1.lbry.com:50001']
{
  "lbryum_servers": []
}
```

## System Configuration

 

- LBRY Daemon version:
- LBRY App version:
- LBRY Installation ID:
- Operating system:

### Acceptance Criteria
1.
2.
3.

### Definition of Done
- [ ]  Tested against acceptance criteria
- [ ] Tested against the assumptions of user story
- [ ] The project builds without errors
- [ ] Unit tests are written and passing
- [ ] Tests on devices/browsers listed in the issue have passed
- [ ] QA performed & issues resolved
- [ ] Refactoring completed
- [ ] Any configuration or build changes documented
- [ ] Documentation updated
- [ ] Peer Code Review performed

**Comment by eukreign** [comment]
This does work via `lbrynet start --lbryum_servers=` and with `settings_set` when done via JSON-RPC. It's only broken via CLI directly. I think this is very low priority issue (since it works in the most common cases).

**Comment by eukreign** [comment]
fixed in #2698
